setup.py

```diff
@@ -195,7 +195,7 @@ def setup_package():
                                 name='refnx.reduce._cevent',
                                 sources=['src/_cevent.pyx'],
                                 include_dirs=[numpy_include],
-                                language='c',
+                                language='c++',
```

Comment:
This is due to a possibly local issue with my gcc compiler/cython version where the compilation would fail as a result of the -> notation that being present in the generated .c file.

Reply:
Interesting. Is that on macOS or linux? I wonder why this hasn't come up on travis?

Reply:
this is linux (ubuntu) with:
- cython 0.28.5
- numpy 1.15.0
- gcc 5.4.0

Reply:
oh and python 3.7.0